arangod/Cluster/DBServerAgencySync.cpp

```diff
@@ -298,16 +346,15 @@ DBServerAgencySyncResult DBServerAgencySync::execute() {
       }
 
       if (tmp.ok()) {
+        VPackSlice planSlice = report.get("Plan");
+        VPackSlice currentSlice = report.get("Current");
         result = DBServerAgencySyncResult(
-            true,
-            report.hasKey("Plan") ? report.get("Plan").get("Version").getNumber<uint64_t>() : 0,
-            report.hasKey("Current")
-                ? report.get("Current").get("Version").getNumber<uint64_t>()
-                : 0);
+          true, planSlice.isObject() ? planSlice.get("Index").getNumber<uint64_t>() : 0,
+          currentSlice.isObject() ? currentSlice.get("Index").getNumber<uint64_t>() : 0);
```

Comment:
The new code new looks up Plan/Index and Current/Index, whereas the old code checked Plan/Version and Current/Version.
This is probably intentional, just wanted to highlight this in case it is unintentional.

Reply:
correct is intentional, as versions may remain unchanged in many cases